src/components/fieldset/fieldset.html

```diff
@@ -1,6 +1,38 @@
-<section>
-  <fieldset></fieldset>
-</section>
+<form>
+  <section>
+    <p>
+      <label>
+        <span>Enter Task</span>
+        <abbr title="required">*</abbr>
+      </label>
+      <input type="text" id="task" name="task" />
+    </p>
+
+    <fieldset>
+      <legend>WHEN</legend>
+      <ul>
+        <li>
+          <label for="time_1">
+            <input type="radio" id="date_1" name="Today" value="" />
+            Today
+          </label>
+        </li>
+        <li>
+          <label for="time_2">
+            <input type="radio" id="date_2" name="Tomorrow" value="" />
+            Tomorrow
+          </label>
+        </li>
+        <li>
+          <label for="time_3">
+            <input type="radio" id="date_3" name="Select Date" value="" />
+            Select Date
+          </label>
+        </li>
+      </ul>
+    </fieldset>
+  </section>
+</form>
 <section>
   <p>
     <button class="btn" type="submit">Add Task</button>
```

Comment:
That `type="submit"` won't do much if the element is outside the `form`

Reply:
THX :D